### DISCUSSION TITLE
I cannot get it working

### DISCUSSION THREAD

**Discussion by PiyushKumarDongre** [PlatformCommunity/flysystem-bunnycdn#45] [General]
Hi, 

I have tried to setup this for my Bunny CDN, I followed the steps carefully for Laravel 9, but can't get it working. There must be something that I am missing to configure correctly. Can you please help me set this up.

Thanks

**Comment by PiyushKumarDongre** [comment]
Hi.
Can you please share an example working instance of project, which I can modify with my account, API  details and test if it works.

Thanks

**Reply by sifex** [reply]
Hi Piyush, did you want to jump on a call and we'll see what I can do?

**Reply by PiyushKumarDongre** [reply]
That will be great. Let me know how to connect.

**Reply by sifex** [reply]
Because I have a flight to catch, post as much detail of the problem you're coming across here in Github.

**Reply by PiyushKumarDongre** [reply]
I was creating Twitch account, We can be done in 1 min or so

**Reply by PiyushKumarDongre** [reply]
Nevermind, I will share my setup detail here.

**Reply by sifex** [reply]
Still please post here
